### ISSUE TITLE
fn.tz keepTime argument is not documented.

### ISSUE THREAD

**Issue by kokeshii** [moment/momentjs.com#586]
The functionality added in https://github.com/moment/moment-timezone/pull/505 is not mentioned anywhere in the documentation.

**Comment by mj1856** [comment]
Indeed.  The optional second parameter to the `tz` instance function is not documented.  It's behavior is similar to the optional second parameter to the `utcOffset` instance function, so the wording should match closely.

See also https://stackoverflow.com/a/56957999/634824
